### DISCUSSION TITLE
What's new in 20.x?

### DISCUSSION THREAD

**Discussion by sklawren** [i18next/i18next#1588] [General]
There's a new major release version (20), but I can't find a changelog, there are no release notes for any of the 20.x versions ([20.0.0](https://github.com/i18next/i18next/releases/tag/v20.0.0), [20.1.0](https://github.com/i18next/i18next/releases/tag/v20.1.0), [20.2.0](https://github.com/i18next/i18next/releases/tag/v20.2.0)), and there's no [migration guide](https://www.i18next.com/misc/migration-guide) for going from v19 to v20 in the official documentation.

Can you please provide some insight into what the breaking changes are?

**Comment by adrai** [comment]
https://www.i18next.com/misc/migration-guide#v-19-x-x-to-v-20-0-0
